autosubmit/job/job.py

```diff
@@ -2479,7 +2478,7 @@ def write_total_stat_by_retries(self, total_stats, first_retrial = False):
         else:
             f.write('\n' + date2str(datetime.datetime.fromtimestamp(total_stats[0]), 'S') + ' ' + date2str(datetime.datetime.fromtimestamp(total_stats[0]), 'S') + ' ' + date2str(datetime.datetime.fromtimestamp(total_stats[1]), 'S') + ' ' + total_stats[2])
         out, err = self.local_logs
-        path_out = os.path.join(self._tmp_path, 'LOG_' + str(self.expid), out)
+        # path_out = os.path.join(self._tmp_path, 'LOG_' + str(self.expid), out)
```

Comment:
If a variable is not used, I think it's better to remove it, instead of leaving a comment (as we can always use git to look at the previous versions of the file where the variable was used).